### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by techsavvyumer** [csom/PlayOnLinux-Office-365#1]
Please guide properly that how to install crossover wine in POL.

**Comment by csom** [comment]
Hi @mumer012001,

## Compile CX-Wine  
*(Hint build Wine on a clean 32-bit virtual machine, I used 32-bit Debian 11 (Bullseye) when building.)

- **At first:** download Codeweavers CrossOver Sources from: [https://media.codeweavers.com/pub/crossover/source/](https://media.codeweavers.com/pub/crossover/source/)
- Unpack the file corresponding to the version of chosing, in this case: **crossover-sources-22.0.0.tar.gz**
- **Second:** follow the guide at WineHQ how to compile Wine for 32-bit architecture: [https://wiki.winehq.org/Building_Wine](https://wiki.winehq.org/Building_Wine)
- **Third:** Create an archive with the compiled files:
```
mkdir /tmp/wine-crossover /tmp/wine-crossover/lib /tmp/wine-crossover/share
cp -R /usr/local/bin "dir-of-extracted-source"/sources/wine/include /tmp/wine-crossover
cp -R /usr/local/lib/wine /usr/local/lib/libwine* /tmp/wine-crossover/lib
cp -R /usr/local/share/man /usr/local/share/wine /tmp/wine-crossover/share
cd /tmp/wine-crossover
tar -zcf cx-wine.tar.gz ./*
```

## Use Compiled CX-Wine with POL

- **Fourth:** - Extract the files in: **~/.PlayOnLinux/wine/linux-x86/cxx-22**

## Install Trial version of CrossOver

- **Fifth:** Install CrossOver 22
- **Sixth:** Create an empty Windows 7 32-bit bottle inside CrossOver _(This is just to get all the requirements for your system)_
- After this it is possible delete CrossOver and the newley created bottle because this is not needed anymore

## Install Microsoft Office 365
- **Last:** Run the script inside POL: [https://github.com/csom/PlayOnLinux-Office-365/blob/cx-7.7/o365.sh](https://github.com/csom/PlayOnLinux-Office-365/blob/cx-7.7/o365.sh)

**Comment by techsavvyumer** [comment]
Hi, thanks for replying, But the crossover 22 source file "crossover-sources-22.0.0.tar.gz" is not available in the mentioned link. This is the link to get the source file: [crossover-sources-22.0.0.tar.gz](https://media.codeweavers.com/pub/crossover/source/crossover-sources-22.0.0.tar.gz).

**Comment by techsavvyumer** [comment]
@csom We need to use 32bit virtual machine just for building the crossover wine, After that we can move this file to over host os (in my case Fedora 36 x64bit) and then move forward with the Fourth step???

**Comment by csom** [comment]
@mumer012001, yes the thing is that it's much easier to use a dedicated 32-bit machine when building Wine if your purpouse is to only use a 32-bit Wine. Which is the case when using with Office 365. You can choose your flavour and dist like you want for your build-system ex. Fedora. I choosed Debian since I use Ubuntu 22.04 for my personal computer. When your wine is build and packed you can use that on x64bit architecture as well.

Edit: It's much easier to get the build dependencies right for :i386 if you use a dedicated 32-bit system.

**Comment by techsavvyumer** [comment]
@Hi, I tried 2,3 times but failed to compile/build wine 32bit in Fedora 25 32bit VM. Is there any other way to build wine?

**Comment by csom** [comment]
Do you get any error message? And if so, what is it saying?

**Comment by csom** [comment]
For me it worked with Debian 11 machine 32-bit. First you need to install all dependencies.
